app/controllers/admin/organizations_controller.rb

```diff
@@ -8,6 +8,11 @@ class OrganizationsController < Admin::ApplicationController
     #   @resources = Organization.all.paginate(10, params[:page])
     # end
 
+    def update
+      super
+      @requested_resource.touch(:profile_updated_at)
```

Comment:
I tested this and it still properly redirects back to `/admin/organizations/:id`, although I didn't think it would. 🤔